Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'moab-versioning', '~> 6.0'
 
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'erb_lint', '~> 0.5.0', require: false
```

Comment:
I see we are currently inconsistent w/r/t how tightly to pin `erb_lint` (argo pins to patch, h2 does not pin at all), so this is at least consistent with Argo!

Reply:
i left this as-is in this PR for now (because that is consistent with argo, and consistent with the laziness-based response to the other bit of feedback on this PR 😆).  but i also added as a short topic for the next dev planning meeting, because this seems worthy of a bit of discussion and standardization.